### ISSUE TITLE
Building r2ghidra on Termux fails due to lack of sudo

### ISSUE THREAD

**Issue by TFullerene** [radareorg/r2ghidra#48]
Hello, after running:
```bash
r2pm init
r2pm update
r2pm -ci r2ghidra
```

I have the following error:

```bash
[ 99%] Linking CXX shared library asm_ghidra.so   
[ 99%] Built target asm_ghidra                    
17 warnings generated.                            
[100%] Linking CXX shared library anal_ghidra.so  [100%] Built target anal_ghidra                   
/data/data/com.termux/files/usr/bin/r2pm: 18: sudo: not found
```
r2ghidra was able to install with previous versions, but from doing a bit of research, termux does not have sudo out of the box. Any help would be greatly appreciated and I would be happy to provide some assistance with this bug. 
Thank you.

**Comment by apkunpacker** [comment]
```sh
pkg install tsu
```
might solve your problem

**Comment by TFullerene** [comment]
Hello, @apkunpacker 
I have reinstalled termux, and installed tsu first before running the r2ghidra installation commands. 
I now have the following message:
```bash
No superuser binary detected.
Are you rooted?
```
Previously I was able to run r2ghidra without having to root my mobile device, however given the error, it may not be possible without first rooting a device. 
Thank you for your recommendation.
